helm/reana/templates/reana-config.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: ui-config
+  name: reana-config
   namespace: {{ .Release.Namespace }}
 data:
   ui-config.yaml: |
```

Comment:
Question: shouldn't we also change `ui-config.yaml`?

Reply:
No, because that file is purely dedicated to the UI. The reason behind this change is that this configmap can have more than one file, so it doesn't make sense to call it ui-config as the files can fulfill different purposes. 

This is kind of a leftover from an attempt to have a `scheduler-config.yaml`, so admins could customize max priorities, ranges and so on, but in the end we decided to go for a different approach based on bins.